e2e/thin_provisioning_test.go

```diff
@@ -105,7 +102,7 @@ func testThinProvisioning() {
 		By("deploying multiple PVCS with total size < thinpoolsize * overprovisioning")
 		// The actual thinpool size is 4 GB . With an overprovisioning limit of 5, it should allow
 		// PVCs totalling upto 20 GB for each node
-		nodeName := "topolvm-e2e-worker2"
+		nodeName := "topolvm-e2e-worker"
```

Comment:
We create a namespace for each `It` and delete the namespace after the spec of the `It` ends. Therefore, it is ok to use the same node.